packages/gasket-plugin-nextjs/lib/next-route.js

```diff
@@ -30,7 +30,7 @@ async function getNextRoute(gasket, req) {
   const routes = await loadRoutes(gasket);
   if (routes) {
     for (const route of iterateRoutes(routes)) {
-      if (route.regex.test(req.url)) {
+      if (route.regex.test(req.path)) {
```

Comment:
One thing we need to check on is if `req.path` is valid for both Fastify and Express.

Reply:
Is it not available in Fastify through [Node](https://nodejs.org/docs/latest-v18.x/api/http.html#requestpath)?

Reply:
Ah, I thought that was an express extension 👍